api/src/main/resources/workbench-api.yaml

```diff
@@ -2479,27 +2482,6 @@ paths:
           description: Successfully added/updated
           schema:
             "$ref": "#/definitions/WorkspaceResource"
-  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/user-recent-resources/delete":
```

Comment:
This function is no longer used and is also not compatible with Swagger3, because of the body in a delete request.